numpy/core/src/multiarray/nditer_constr.c

```diff
@@ -2987,14 +2966,6 @@ npyiter_allocate_arrays(NpyIter *iter,
                         "but neither copying nor buffering was enabled");
                 return 0;
             }
-
-            /*
-             * If the operand is aligned, any buffering can use aligned
-             * optimizations.
-             */
-            if (IsUintAligned(op[iop])) {
-                op_itflags[iop] |= NPY_OP_ITFLAG_ALIGNED;
-            }
```

Comment:
Are there "aligned optimizations" (as mentioned in the comment) beyond the call to `PyArray_GetDTypeTransferFunction`?